README.md

```diff
@@ -1,2 +1,133 @@
-# carbonite-migrate
-Migrate devices using Carbonite Migrate.
+carbonite-migrate
+=================
+
+# Migrate devices using Carbonite Migrate.
+ 
+User requirements
+=================
+
+**The following software must be running/installed on the client computer**:
+
+1.  Device42 16.08.03+
+
+2.  Carbonite console 8.3.0.293+
+
+3.  **After installing the Carbonite Console, please copy the
+    "DoubleTake.PowerShell.dll" from its install directory (by default in
+    C:\\Program Files\\Carbonite\\Replication\\Console) to the root folder of
+    your scripts' location!**
+
+4.  PowerShell 5.1: if you are using Windows 10, you already have the necessary
+    version of PowerShell, if you are using Windows 7/8/8.1; please download the
+    Windows Management Framework 5.1 that includes the necessary updates to
+    Windows PowerShell -\> [Download WMF
+    5.1](https://www.microsoft.com/en-us/download/details.aspx?id=54616)
+
+5.   For convenience, you can use PowerShell ISE, which includes a dual screen
+    layout where you can view the script and run it at the same time using a
+    GUI. Run it by pressing the Windows key and typing “powershell ise”.
+
+    ![PS ISE](https://i.imgur.com/uFcC013.png)
+
+6.  If you never ran a PowerShell script before, you will need to perform the
+    following steps:
+
+    -   Start Windows PowerShell with the "Run as Administrator" option. Only
+        members of the Administrators group on the computer can change the
+        execution policy.
+
+    -   Enable running unsigned scripts by entering: `Set-ExecutionPolicy
+        RemoteSigned`
+
+    ![alt](https://i.imgur.com/WqSTevh.png)
+
+Creating a CSV export for Carbonite Migration
+=============================================
+
+1.  Access the D42 website and select Apps \> Business Application from the main
+    toolbar.
+
+![alt](https://i.imgur.com/3lv9tKf.jpg)
+
+2.  On the next screen, add a desired business application or select an existing
```

Comment:
This should be updated since you can now select one or more business applications.

Reply:
Done